hw/ip/sensor_ctrl/lint/sensor_ctrl.vlt

```diff
@@ -0,0 +1,5 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+//
+// waiver file for rstmgr
```

Comment:
nit: copy-paste typo (here and below file)

Reply:
doh! thanks for the catch. Fixed in #2273